test/IntegrationTests/SqlTests.cs

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 // </copyright>
 
+#if !NETFRAMEWORK
```

Comment:
Instead of doing this here and in `SqlClientCollection.cs` we should remove the line:

```csharp
span.Tags["db.statement_type"].Should().Be("Text");
```

The test is failing on .NET Framework, because

> On .NET Framework, "db.statement_type" attribute is no longer set for
  activities created by the instrumentation.

Source: https://github.com/open-telemetry/opentelemetry-dotnet/blob/55260e6858e5ecf691c60161ea25d29bbc24dc1d/src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md?plain=1#L78-L79

Also if possible I suggest refactoring the assertions 

```csharp
span.Tags["db.system"].Should().Be("mssql");
```

into this patttern

```csharp
span.Tags.Should().Contain(new KeyValuePair<string, string>("db.system", "mssql"));
```

It would make the assertion messages a lot clearer.


Reference: https://fluentassertions.com/dictionaries/

Reply:
4b38002